pyproject.toml

```diff
@@ -1,7 +1,7 @@
 [tool.black]
 line-length = 100
 skip-string-normalization = true
-target_version = ['py310']
+target_version = ['py37', 'py38']
```

Comment:
@ametzger as noted in slack, I'm rolling this back. I think we need to update `black` before we can use it with py310?